### DISCUSSION TITLE
RPi 4B GPIO pinout to the ULN2003 driver for 28BYJ-48 stepper?

### DISCUSSION THREAD

**Discussion by astrotater** [aaronwmorris/indi-allsky#1330] [General]
I'm wrapping up an enclosure that incorporates the 28BYJ-48 stepper driver for focusing, and an expansion PCB for I/O with space for the ULN2003 driver. 
What are the indi-allsky pinouts to connect to IN1, IN2, IN3, IN4 of the ULN2003 driver board?

**Comment by aaronwmorris** [comment]
The pin numbers are completely configurable in indi-allsky.  The pins I used in my test setup are the same ones in the examples of the config form...

```
IN1 = D17
IN2 = D18
IN3 = D27
IN4 = D22
```

**Reply by astrotater** [reply]
Thanks!
